src/colorizer/loaders/JsonArrayLoader.ts

```diff
@@ -1,6 +1,9 @@
+import { parquetRead } from "hyparquet";
```

Comment:
I will rename this file to `UrlArrayLoader` once reviews come in! Leaving this as `JsonArrayLoader` so it's easier to diff.